src/app/components/chat/window/output/item/item.vue

```diff
@@ -103,7 +108,26 @@
 					</span>
 				</template>
 
-				<app-content-viewer :source="message.content" :display-rules="displayRules" />
+				<template v-if="message.type === ChatMessage.TypeNormal">
+					<app-content-viewer :source="message.content" :display-rules="displayRules" />
+				</template>
+				<template v-else-if="message.type === ChatMessage.TypeInvite">
+					<div class="alert">
+						<p>
+							<translate>
+								You have been invited to a group.
+							</translate>
+						</p>
+						<app-button
+							:disabled="chat.currentUser && message.user.id === chat.currentUser.id"
```

Comment:
I think instead of just disabling the accept button for the sending user, we should show a completely different message to the sender:
Have the text say something like "You have invite this user to a group".
I am not sure if we want to support canceling an invite, but that button would also go here I guess.

Reply:
Also, make that check a getter on the component class, something like `isInviteSender`.